src/main/java/uk/gov/digital/ho/hocs/search/application/RequestData.java

```diff
@@ -98,4 +80,8 @@ public String groups() {
         return MDC.get(GROUP_HEADER);
     }
 
+    private static boolean isNullOrEmpty(String value) {
```

Comment:
Better use in a StringUtils class, I have 0 doubts that this could be used elsewhere...